codegen/scala-gen/src/main/scala/kalix/codegen/scalasdk/impl/ActionTestKitGenerator.scala

```diff
@@ -137,12 +137,12 @@ object ActionTestKitGenerator {
           |
           |  "${actionClassName}" must {
           |
-          |    "have example test that can be removed" in {
-          |      val testKit = ${actionClassName}TestKit(new $actionClassName(_))
+          |    "have example test that can be removed" pending {
+          |      val service = ${actionClassName}TestKit(new $actionClassName(_))
           |      // use the testkit to execute a command
```

Comment:
Shall we have a TODO comment here to hint that this test can be implemented based on the following template and that the `pending` should be removed?